RELEASE_NOTES.md

```diff
@@ -1,5 +1,51 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# What's New in 16.4
+
+## Resource Updates
+
+### chef_client_systemd_timer
+
+The `chef_client_systemd_timer` has been updated to prevent failures running the `:remove` action.
```

Comment:
```suggestion
The `chef_client_systemd_timer` resource has been updated to prevent failures running the `:remove` action.
```

Reply:
I feel like this might be missing a word, instead of `failures`, something like `failed runs`?